engine/CMakeLists.txt

```diff
@@ -1238,13 +1248,21 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
 			MESSAGE("Still need to generate dependency list")
 		ENDIF (EXISTS "${CMAKE_BINARY_DIR}/dependency.list")
 		# Alternatively, just set a hard-coded set of dependencies...requires more logic for various platforms.
-		#IF (USE_DEBIAN_VERSION)
-		#	# Debian Dependency Chain
-		#	SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python2, libjpeg, libpng, freeglut, gtk2, libvorbis, openal, SDL_mixer, xdg-utils, opengl-games-utils")
-		#ELSE (USE_DEBIAN_VERSION)
-		#	# Ubuntu Dependency Chain
-		#	SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python2, libjpeg62, libpng16-16, freeglut, gtk2, libvorbis, openal, SDL_mixer, xdg-utils, opengl-games-utils")
-		#ENDIF (USE_DEBIAN_VERSION)
+		IF (USE_DEBIAN_VERSION)
+			# Debian Dependency Chain
+            IF (DEBIAN_RELEASE_VERSION STREQUAL "buster")
+                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python, libjpeg62-turbo, libpng16-16, freeglut3, libgtk2.0-0, libvorbis0a, libopenal1, libsdl-mixer1.2, libsdl-gfx1.2-5, xdg-utils, libgl1, libboost-python1.67.0")
+            ENDIF ()
+		ELSE (USE_DEBIAN_VERSION)
+			# Ubuntu Dependency Chain
+            IF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "xenial")
+                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python, libjpeg62, libpng16-16, freeglut3, libgtk2.0-0, libvorbis0a, libopenal1, libsdl-mixer1.2, libsdl-gfx1.2-5, xdg-utils, libgl1-mesa-glx, libboost-python1.58.0")
+            ELSEIF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "bionic")
+                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python, libjpeg62, libpng16-16, freeglut3, libgtk2.0-0, libvorbis0a, libopenal1, libsdl-mixer1.2, libsdl-gfx1.2-5, xdg-utils, libgl1, libboost-python1.65.1")
+            ELSEIF (LSB_LINUX_DISTRIBUTION_CODENAME STREQUAL "focal")
+                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "python2, libjpeg62, libpng16-16, freeglut3, libgtk2.0-0, libvorbis0a, libopenal1, libsdl-mixer1.2, libsdl-gfx1.2-5, xdg-utils, libgl1, libboost-python1.67.0")
+            ENDIF ()
+		ENDIF (USE_DEBIAN_VERSION)
```

Comment:
This is actually auto-generated by the script - it takes the binary and looks up all dynamic libraries to determine what the package dependencies really are instead of relying on something static that we do right now.

Reply:
Also, did you use tabs instead of spaces? There seems to be a few things that don't line up properly.

Reply:
I used spaces; someone else used tabs. :-P I'll go ahead and fix those though.